src/main/resources/stemmer.yml

```diff
@@ -228,6 +237,7 @@ protected_words:
   - ΦΙΣ
   - ΦΛΟΤΕΡ
   - ΦΠΑ
+  - ΧΑΡΤΗ
```

Comment:
[minor] why not just add another `OR` clause?

Reply:
It's not the same case with the other 3 (they're having an extra letter) - we would need an extra check and I preferred the protection to this.